### ISSUE TITLE
AccesibilityInfo

### ISSUE THREAD

**Issue by eschablowski** [necolas/react-native-web#698]
**Do you want to request a *feature* or report a *bug*?**
bug
**What is the current behavior?**
Throws;  Cannot find module 'AccessibilityInfo' from <react-native>

1.https://github.com/eschablowski/morscout-client

**What is the expected behavior?**
No Error
**Environment (include versions). Did this work in previous versions?**

* OS: Ubuntu 17.10
* Device: Laptop
* Browser: Chrome
* React Native for Web (version): 0.1.12 (installed with npm)
* React (version): 16.0.0 (also installed with npm)

buildfile:
`let browserify = require("browserify");
let fs = require("fs");
let path = require("path");
let dir = path.relative(process.cwd(), path.join(__dirname, "../"));
dir = dir != "" ? dir : ".";
require("./create_page.js");
require("./cleanFolders")("web", dir);
let libs = [
    "axios",
    "dompurify",
    "radium",
    "uranium",
    "react",
    "react-native",
    "socket.io-client",
];
if (fs.existsSync(path.join(dir, "Page.js"))) {
    var browser = browserify(path.join(dir, "Page.js"))
    for (lib in libs)
        browser.external(lib);
    browser.plugin(require("tsify"), { noImplicitAny: true })
    browser.transform("babelify", {
        "plugins": [
            ["babel-plugin-root-import", [
                {
                    rootPathPrefix: "~",
                    rootPathSuffix: "src"
                },
                {
                    rootPathPrefix: "$",
                    rootPathSuffix: "morscout-api/javascript-browser"
                },
                {
                    rootPathPrefix: "#",
                    rootPathSuffix: "style"
                },
                {
                    rootPathPrefix: "@",
                    rootPathSuffix: "web/Components"
                }
            ], "transform-decorators-legacy", "react-native-web/babel"]
        ],
        "presets": [
            [
                "env",
                {
                    "targets": {
                        "browsers": [
                            "last 2 versions",
                            ">1%"
                        ]
                    }
                }
            ],
            "react", "stage-0"
        ],
    })
        /*.transform('uglifyify', {
            global: true,
            output: {
                ascii_only: true,
            },
            ecma: 8,
            toplevel: true,
            compress: true,
        })*/
        .bundle()
        .pipe(fs.createWriteStream(path.join(dir, "build", "web", "Page.js")));
    fs.readdir(path.join(dir, "web"), (err, files) => {
        if (err) {
            console.error("Error " + err.code + ": " + err.message);
        }
        for (var file in files) {
            file = files[file];
            if (fs.statSync(path.resolve(process.cwd(), dir, "web", file)).isFile())
                fs.createReadStream(path.resolve(process.cwd(), dir, "web", file))
                    .pipe(fs.createWriteStream(path.resolve(process.cwd(), dir, "build", "web", file)));
        }
    });
}`

**Comment by eschablowski** [comment]
Reinstalled from github and worked
